File: official/docs/node/v6/pickups/cancel.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const pickup = await client.Pickup.retrieve('pickup_...');

  const cancelledPickup = pickup.cancel(pickup.id);

  console.log(cancelledPickup);
})();

File: official/docs/csharp/v4/pickups/cancel.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.Pickup pickup = await client.Pickup.Retrieve("pickup_...");

            await pickup.Cancel();

            Console.WriteLine(JsonConvert.SerializeObject(pickup, Formatting.Indented));
        }
    }
}
